src/Adapters/Cache/InMemoryCache.js

```diff
@@ -0,0 +1,70 @@
+const DEFAULT_CACHE_TTL = 60 * 1000;
```

Comment:
Anyway to override this value with an ENV variable ou CLI option or Config value ?